### ISSUE TITLE
Error: Cannot find module 'uuid'

### ISSUE THREAD

**Issue by patriq** [giorgi-o/SkinPeek#39]
Tried using the docker image `ghcr.io/giorgi-o/skinpeek/skinpeek` using the docker-compose file in the repository, and it looks like node-cron needs uuid but can't find it?

```
      

node:internal/modules/cjs/loader:933


  const err = new Error(message);


              ^



Error: Cannot find module 'uuid'


Require stack:


- /usr/app/node_modules/node-cron/src/scheduled-task.js


- /usr/app/node_modules/node-cron/src/node-cron.js


    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)


    at Function.Module._load (node:internal/modules/cjs/loader:778:27)


    at Module.require (node:internal/modules/cjs/loader:999:19)


    at require (node:internal/modules/cjs/helpers:102:18)


    at Object.<anonymous> (/usr/app/node_modules/node-cron/src/scheduled-task.js:6:14)


    at Module._compile (node:internal/modules/cjs/loader:1099:14)


    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)


    at Module.load (node:internal/modules/cjs/loader:975:32)


    at Function.Module._load (node:internal/modules/cjs/loader:822:12)


    at Module.require (node:internal/modules/cjs/loader:999:19) {


  code: 'MODULE_NOT_FOUND',


  requireStack: [


    '/usr/app/node_modules/node-cron/src/scheduled-task.js',


    '/usr/app/node_modules/node-cron/src/node-cron.js'


  ]


}



Node.js v17.9.1

      
      
```

**Comment by patriq** [comment]
Yeah seems like node-cron is having the same issue: https://github.com/node-cron/node-cron/issues/355

**Comment by patriq** [comment]
Could you maybe fix the node-cron to 3.0.0?

**Comment by giorgi-o** [comment]
yup! fixed in https://github.com/giorgi-o/SkinPeek/commit/17e96d1c834bc7109d49101917b341a1e4225e0f.
